spec/integ/bridge.spec.js

```diff
@@ -187,6 +187,22 @@ describe("Bridge", function() {
             });
```

Comment:
I am seeing linting and test errors for something I didn't change. Am I doing something wrong?

https://buildkite.com/matrix-dot-org/matrix-appservice-bridge/builds/411#dad87f0f-d7c1-4a2c-a9e6-8e9ea816d86d
```
src/components/client-factory.ts:128:62 - error TS2731: Implicit conversion of a 'symbol' to a 'string' will fail at runtime. Consider wrapping this expression in 'String(...)'.
--
  |  
  | 128                 const loggerInstance = logging.get(`js-sdk:${loggerName}`);
```

Reply:
This is https://github.com/matrix-org/matrix-appservice-bridge/pull/286